src/scripts.js

```diff
@@ -54,15 +60,15 @@ const displayCurrentRecipes = () => {
   mainDisplay.innerHTML = '';
   display.forEach(recipe => {
     mainDisplay.innerHTML +=`
-    <article class="flex column sml-brdr-radius shadow clickable" id="${recipe.id}">
-      <img class="full-width half-height not-clickable" src=${recipe.image}>
-      <div class="flex row around full-width half-height yellow not-clickable">
+    <article class="flex column sml-brdr-radius shadow clickable">
+      <img class="full-width half-height recipe-image" id="${recipe.id}" src=${recipe.image}>
+      <div class="flex row around full-width half-height yellow">
```

Comment:
So relieved we figured out the old not-clickable portion was the piece preventing all our eventListeners from working!